deepchecks/core/serialization/check_result/html.py

```diff
@@ -291,6 +295,35 @@ def handle_figure(cls, item, index, **kwargs) -> str:
             validate=True,
         )
 
+    @classmethod
+    def handle_display_map(cls, item: 'check_types.DisplayMap', index: int, **kwargs) -> str:
+        """Handle display map instance item."""
+        template = """
+            <details>
```

Comment:
nice didn't know about details tag

Reply:
yup) I also learned about it only recently